.deploy/gunicorn.conf.py

```
@@ -4,13 +4,48 @@
 bind = "0.0.0.0:9090"
-workers = 2
+
+# (2 * CPU) + 1 is the standard formula for I/O-bound gthread workers.
+# No CPU limit is set on the container, so this scales to whatever the host
+# provides (2-core → 5 workers, 4-core → 9 workers, etc.).
+workers = multiprocessing.cpu_count() * 2 + 1
+
 worker_class = "gthread"
+
+# 4 threads per worker is right for this I/O-bound API (Postgres + Dragonfly).
+# Total concurrent DB connections = workers × threads; keep that well below
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)